docs/current_docs/ci/quickstart/env.mdx

```diff
@@ -65,22 +65,26 @@ If you've worked with Dockerfiles, this Dagger Function should look very familia
 Now that you know how the Dagger Function works, call it as below:
 
 ```shell
-dagger call build-env --source=.
+dagger -c 'build-env .'
```

Comment:
From the perspective of a new user, I think this command (without the `source` arg) is harder to grok/mentally map to the function shown previously. It would be great if we could support `dagger -c 'build-env --source=.' as well as the above.

Reply:
Another point to consider: if there are a number of required arguments, it's hard to figure out what each one represents without the argument name. This is visible when you compare the old and new chaining examples in https://github.com/dagger/dagger/pull/9811 .

Reply:
Discussion here:
- https://github.com/dagger/dagger/issues/9701

Reply:
I know you want to "simplify" the build function later in the tutorial, but if `source` had a default from the beginning, it would remove this CLI argument problem. And also, it would make the example more realistic from the start. So, I propose that we do that.

Reply:
@shykes I merged this PR before seeing your comment above so will open a separate PR for this change.

Reply:
Just realized: this is more than a code change, as all the screen recordings are affected (since they all show the `source` argument in use)

Reply:
This change in progress at https://github.com/dagger/dagger/pull/9937